inventory/group_vars/_example.yml

```diff
@@ -10,3 +10,9 @@ language_packages:
   - language-pack-en-base
 l10n_repo: https://github.com/openfoodfoundation/l10n_au.git
 timezone:
+
+developer_email: <dev email>
```

Comment:
Some comments about the purpose would be nice. And how to keep this private if someone doesn't want to publish their email address.